specification/metadata/focusversion.md

```diff
@@ -0,0 +1,17 @@
+# FOCUS Version
+
+The declared FOCUS version of the FOCUS data within the schema.
```

Comment:
```suggestion
The version of FOCUS utilized for building the dataset.
```

Reply:
Here I think we need to be mindful that it's not necessarily conveying compliance. I assume many providers will likely not be 100% compliant but will provide data starting with a particular FOCUS version and incrementally add support.

Reply:
Yeah that is why I have this as a metadata signifying the version declared in that schema. If this is saying that they are following a specific version in general, the problem is that how does the practitioner know the version for older data. That is why I have it attached to the schema itself.